File: student/src/main/java/nju/classroomassistant/student/ui/discussion/DiscussionActivity.java
package nju.classroomassistant.student.ui.discussion;

import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import nju.classroomassistant.student.R;

public class DiscussionActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_discussion);
        setActionBar();
    }

    private void setActionBar() {
        ActionBar actionBar = getSupportActionBar();
        if (actionBar == null)
            return;
        actionBar.setDisplayHomeAsUpEnabled(true);
        actionBar.setDisplayShowHomeEnabled(false);
        actionBar.setTitle(R.string.discussion_inactive);
        actionBar.setDisplayShowTitleEnabled(true);
    }

    @Override
    public boolean onSupportNavigateUp() {
        finish();
        return super.onSupportNavigateUp();
    }

}

File: student/src/main/java/nju/classroomassistant/student/views/discussion/DiscussionActivity.kt
package nju.classroomassistant.student.views.discussion

import android.os.Bundle
import android.app.Activity
import androidx.appcompat.app.AppCompatActivity
import nju.classroomassistant.student.R

import kotlinx.android.synthetic.main.activity_discussion.*
import nju.classroomassistant.student.systemstate.SystemState
import java.util.*

class DiscussionActivity : AppCompatActivity(), Observer {
    override fun update(p0: Observable?, p1: Any?) {

    }

    private fun syncState() {
        val started = SystemState.discussion.started


    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_discussion)
        actionBar?.setDisplayHomeAsUpEnabled(true)


    }

}
